2-mandatory/3-financial-times.js

```diff
@@ -23,59 +33,81 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+  let popularHeadline = [];
+  for (let i in allArticleTitles) {
+    if (allArticleTitles[i].match(/[0-9]/g) !== null) {
```

Comment:
Great use of the `[0-9]` regular expression, you can also shortcut this to `\d` which means decimals but what you did is probably more readable! Great job 💯